src/tools/ResultsComparer/CommandLineOptions.cs

```diff
@@ -29,6 +29,9 @@ public class CommandLineOptions
         [Option("csv", HelpText = "Path to exported CSV results. Optional.")]
         public FileInfo CsvPath { get; set; }
 
+        [Option('f', "filter", HelpText = "Filter the benchmarks by name using glob pattern(s). Optional.")]
+        public IEnumerable<string> Filters { get; set; }
```

Comment:
What's the scenario for passing multiple filters?

Reply:
> What's the scenario for passing multiple filters?

When you own two things in CoreFX that are similar but don't belong to the same namespace. Examples:

```cmd
--filter System.Math* System.Numerics*
```

```cmd
--filter System.Collections* *LINQ*
```